src/you_get/processor/ffmpeg.py

```diff
@@ -6,9 +6,18 @@
 from ..util.strings import parameterize
 from ..common import print_more_compatible as print
 
+try:
+    from subprocess import DEVNULL
+except ImportError:
+    # Python 3.2 or below
```

Comment:
Not sure if we need to maintain Python 3.2 compatibility. If not, this compat block can be removed.